packages/web-components/src/components/ic-button/ic-button.tsx

```diff
@@ -64,6 +64,11 @@ export class Button {
   @State() describedByContent: string = null;
   @State() title: string = null;
 
+  /**
+   * The accepted list of files.
```

Comment:
Could you expand this description to explain that it's only valid when file-upload is set to true?